homework_4/numpy_challenge.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+if __name__ == '__main__':
+    # my favourite arrays
+    a1 = np.arange(20).reshape((4, 5))
+    a2 = np.eye(5)
+    a3 = np.empty_like(a1, dtype=object)
+
+
+# true matrix multiplication
+def matrix_multiplication(b1, b2):
+    return np.matmul(b1, b2)
```

Comment:
Что означает `b`?

Мне ещё нравится вот так, но разницы по сути нет
```suggestion
def matrix_multiplication(b1, b2):
    return b1 @ b2
```